fluent/lib/swagger-client.js

```diff
@@ -0,0 +1,232 @@
+'use strict'
```

Comment:
Are we dynamically re-generating the client here? This feels awkward/ugly... Let's integrate with the `kubernetes-client/gen` repo if we want to do generation.

Reply:
Or download and generate on the fly from the swagger hosted on the API server.

Reply:
It's not static codegen, we're generating the object chains on the fly.  There's support for loading swagger specifications included in the npm package, but I'm unsure how popular that is: internally at GoDaddy, we always use the swagger hosted with the API server.

Reply:
... at a higher-level, do you have strong reactions to the on-the-fly-creating-the-client approach?

Reply:
I don't have super-strong objections, but it is definitely different than what we do with all other clients.

The only concrete concern I have with this approach is that it tends to make library load quite slow (as the client is generated) so we should definitely not require this for at least the non-fluent parts of the library that don't require it...